parity/cli.rs

```diff
@@ -155,6 +155,7 @@ Footprint Options:
   --cache MEGABYTES        Set total amount of discretionary memory to use for
                            the entire system, overrides other cache and queue
                            options.
+  --db-cache-size MEGABYTE Database cache size.
```

Comment:
Perhaps this should be a single  --cache options with 20% going to blockchain cache and 80% to db cache, actual ratio needs research though